queries/php/highlights.scm

```diff
@@ -62,6 +62,9 @@
 (object_creation_expression
   [(name) (qualified_name)] @constructor)
 
+(argument
+    (name) @constructor)
```

Comment:
Any reason for this change? This is not done for Python named arguments.

Reply:
I don't know much about syntax highlighting in Python but, looking VSCode way of highlighting named arguments in various themes I thought would be cool to have this highlight

VSCode- Community Material Theme Palenight
![image](https://user-images.githubusercontent.com/20175215/154819700-8e1590ef-2a15-41f2-b218-657d2efd266c.png)

Neovim - Palenight colorscheme _**without** this PR_
![image](https://user-images.githubusercontent.com/20175215/154819803-088aab0b-6087-4f47-a0b4-b303af454bc0.png)

Neovim - Palenight colorscheme _**with** this PR_
![image](https://user-images.githubusercontent.com/20175215/154819961-aa86c169-3003-458e-8c41-73364c8a8a0f.png)

Reply:
but why not use `@parameter` for this highlight. All other usages of `@constructor` are for the type being constructed (alias the constructor call)

Reply:
Hmm, I guess since this can only be used on constructors...

Reply:
Python uses `@parameter` but it can also be used in normal function calls and normally `@parameter` is only used at the callee site... let me think a bit about that...

Reply:
>Hmm, I guess since this can only be used on constructors...

Same here

Reply:
@kyazdani42 @stsewd what do you think about which highlight is best for this?

Reply:
i don't have a strong opinion about this, but i think `@parameter` makes more sense when it's a named function parameter. Using constructor is not really appropriate in this scenario though.

Reply:
would agree with that. @wilmanbarrios can you change to `@parameter`?

Reply:
Yes sure, later today I will make de change :+1: